examples/platform_view/ios/Runner/Base.lproj/Main.storyboard

```diff
@@ -49,7 +49,7 @@
                                                 <color key="textColor" white="1" alpha="1" colorSpace="calibratedWhite"/>
                                                 <nil key="highlightedColor"/>
                                             </label>
-                                            <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="K7X-Ru-0N7" customClass="MDButton">
```

Comment:
Removed buttonType="roundedRect" (deprecated) and customClass="MDButton" (we never defined this in the first place).